packages/trader/src/App/Components/Form/CompositeCalendar/composite-calendar-mobile.jsx

```diff
@@ -1,7 +1,7 @@
 import classNames from 'classnames';
 import PropTypes from 'prop-types';
 import React from 'react';
-import { Icon, MobileDialog, Button } from '@deriv/components';
+import { Icon, MobileDialog, Button, Div100vhContainer } from '@deriv/components';
```

Comment:
Can we remove `Div100vhContainer` import since it is not used?

Reply:
Yep, missed this somehow. Thanks